mmv1/products/compute/RegionSecurityPolicy.yaml

```diff
@@ -193,3 +197,495 @@ properties:
             Encoded as a hexadecimal number (starting with "0x").
             The last byte of the field (in network byte order) corresponds to the least significant byte of the mask.
           min_version: 'beta'
+  - name: 'rules'
+    type: Array
+    description: |
+      The set of rules that belong to this policy. There must always be a default rule (rule with priority 2147483647 and match "*"). If no rules are provided when creating a security policy, a default rule with action "allow" will be added.
+    min_version: 'beta'
+    required: false
+    immutable: false
+    default_from_api: true
+    item_type:
+      type: NestedObject
+      properties:
+        - name: 'description'
+          type: String
+          description: |
+            An optional description of this resource. Provide this property when you create the resource.
+          min_version: 'beta'
+        - name: 'priority'
+          type: Integer
+          description: |
+            An integer indicating the priority of a rule in the list.
+            The priority must be a positive value between 0 and 2147483647.
+            Rules are evaluated from highest to lowest priority where 0 is the highest priority and 2147483647 is the lowest priority.
+          min_version: 'beta'
+          required: true
+          immutable: true
```

Comment:
This doesn't seem to be immutable on [SecurityPolicy](https://github.com/hashicorp/terraform-provider-google-beta/blob/4699748cee90655d692077aa3936edf261c82892/google-beta/services/compute/resource_compute_security_policy.go#L88) - are you sure it's immutable here?